### DISCUSSION TITLE
Add ability to search for difficulty names using brackets

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#24853] [Ideas] [answered]
Something that you can do in osu!stable is put a difficulty name in between brackets and the game will show you all of the maps that have a difficulty matching that exact difficulty name.

![image](https://github.com/ppy/osu/assets/74088315/1935b814-f414-41f7-bf84-208e1b13131c)

If I try to type [haze] to search in osu!lazer, I get no matches despite having the map. It's a very useful search filter and it would be cool to see it in osu!lazer.

**Comment by peppy** [comment, marked as answer]
Tracking at #24855.
